wiki/Gameplay/Spectating/de.md

```diff
@@ -0,0 +1,16 @@
+---
+stub: true
+tags:
+  - spec
+  - spectate
+  - spectator
+  - zuschauen
+---
+
+# Zuschauen
+
+<!-- TODO: needs to be combined with /wiki/Replay#spectator in some way -->
+
+**Zuschauen** oder **Spectate** (oft abgekürzt mit "spec") ist ein Begriff, der das Beobachten eines anderen Spielers beim Spielen einer [Beatmap](/wiki/Beatmap) in Echtzeit (mit ca. 500ms Verzögerung) beschreibt. Um zuzuschauen, öffne den erweiterten Chat und klicke auf ein graues Benutzerfeld.
+
+Damit ein Nutzer einen anderen Spieler beobachten kann, muss er genau dieselbe Beatmap heruntergeladen haben, die der zu beobachtende Spieler gerade spielt. Wenn der Spieler, der beobachtet wird, eine neue Beatmap zu spielen beginnt und der Zuschauer diese Beatmap nicht hat, wird die neue Beatmap in einem privaten `#spectator` Kanal angekündigt. Wenn der Zuschauer jedoch einen [osu!supporter](/wiki/osu!supporter) Tag hat, wird die Beatmap automatisch über osu!direct heruntergeladen und im Spiel für ihn verarbeitet.
```

Comment:
```suggestion
Damit ein Nutzer einen anderen Spieler beobachten kann, muss er genau dieselbe Beatmap heruntergeladen haben, die der zu beobachtende Spieler gerade spielt. Wenn der Spieler, der beobachtet wird, eine neue Beatmap zu spielen beginnt und der Zuschauer diese Beatmap nicht hat, wird die neue Beatmap in einem privaten `#spectator` Kanal angekündigt. Wenn der Zuschauer jedoch ein [osu!supporter](/wiki/osu!supporter) Tag hat, wird die Beatmap automatisch über osu!direct heruntergeladen und im Spiel für ihn verarbeitet.
```

I make this mistake way too often myself, it just sounds more natural

[day -> der Tag; tag -> das Tag](https://www.korrekturen.de/genus/der-oder-das-tag.shtml)